### DISCUSSION TITLE
建议：使用 GitHub Actions 自动构建并上传 artifacts 到 release

### DISCUSSION THREAD

**Discussion by amorphobia** [imfuxiao/Hamster#32] [Ideas]
缺点：
- 需要额外写 workflows 的 yaml 配置

优点：
- 在 tag 和 (pre-)release 的时候自动构建并签名，可以自动上传到 testflight 或 app store
- 在 push 的时候自动构建但不签名，可以方便小白鼠自己侧载，更快测试到新版本，也方便没有 mac 的小白鼠可以 fork 后使用自己仓库的 action 构建

**Comment by imfuxiao** [comment]
谢谢你的建议, 我有时间研究下...
